### DISCUSSION TITLE
Renaming files from explorer.

### DISCUSSION THREAD

**Discussion by conjureseizures** [termux/termux-app#3527] [Q&A] [answered]
Hi all. I use a file manager where you can select Termux application storage. In this storage I store all the files that I can work with in Termux. I'm having a problem when I try to directly rename a folder or file from explorer and I get an error. However, I can delete and create files. It's not always convenient for me to rename files through the terminal. Is there any way to fix this? I will be very grateful.

**Comment by agnostic-apollo** [comment, marked as answer]
Will be fixed in next version. #2415

**Reply by conjureseizures** [reply]
When can we expect the next update?

**Reply by agnostic-apollo** [reply]
Can'r provide concrete days, it has already been delayed by a lot. SAF provider also still needs some finalization.
